### PR TITLE
moving files to their correct <role>/files location

### PR DESCRIPTION
**Pull request by yocum137**
The openshift_web_console and template_service_broker roles are putting 'files' outside the openshift-ansible roles they belong to.

### CODE REVIEW

**Comment by yocum137** [comment]
aos-ci-test

**Comment by spadgett** [comment]
Related comment: https://github.com/openshift/openshift-ansible/pull/6359#pullrequestreview-90344219

**Comment by yocum137** [comment]
Chatting with spadgett, seems the plan is to have the TSB template files automatically sync'd from origin via "some process" (githook?).  I contend that they should go directly into the roles/template-service-broker/files/ directory.

**Comment by spadgett** [comment]
LGTM

File: roles/template_service_broker/vars/main.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-__tsb_files_location: "../../../files/origin-components/"
+__tsb_files_location: "../files/"
```
Comment (mtnbikenc) [inline roles/template_service_broker/vars/main.yml]: If the files are moving into `<role>/files`, this var is no longer necessary to explicitly define the path.  The two `copy` tasks using this var should be updated.

http://docs.ansible.com/ansible/latest/playbooks_reuse_roles.html#using-roles
> Any copy, script, template or include tasks (in the role) can reference files in roles/x/{files,templates,tasks}/ (dir depends on task) without having to path them relatively or absolutely.

File: roles/openshift_web_console/vars/main.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-__console_files_location: "../../../files/origin-components/"
+__console_files_location: "../files/"
```
Comment (mtnbikenc) [inline roles/openshift_web_console/vars/main.yml]: Previous comment about role path applies here.

**Comment by sdodson** [comment]
The goal was that these would be sync'd automatically from origin and it'd be easiest if we were just working on one directory tree rather than splitting them across several roles. That automation has never been implemented as other work for both CI/CD and CL teams have taken priority.

Unless this is really seen as a blocker for Ops I don't want to make this change before 3.9.

**Comment by sdodson** [comment]
Back tracking on that a bit, will discuss during our arch call today.

**Comment by sdodson** [comment]
Summary from openshift-ansible arch call is that there's overwhelming support for moving these into the roles that actually use them. So lets move forward with that. They're not currently versioned but we'll want to make sure that they're versioned prior to 3.10.

File: roles/template_service_broker/vars/main.yml (hunk 1 above)
Comment (yocum137) [reply roles/template_service_broker/vars/main.yml]: +1

File: roles/openshift_web_console/vars/main.yml (hunk 2 above)
Comment (yocum137) [reply roles/openshift_web_console/vars/main.yml]: +1

**Comment by yocum137** [comment]
aos-ci-test

**Comment by sdodson** [comment]
See the list of statuses below, tests are running.

**Comment by mtnbikenc** [comment]
"ERROR: Error running command: rpmbuild"
/restest

**Comment by mtnbikenc** [comment]
Ah, this is going to require a specfile update.  Need to back out #5785.

**Comment by sdodson** [comment]
The specfile needs to be updated to remove files from the build/install and files section.

```
+ cp -rp files /data/src/github.com/openshift/openshift-ansible/tito/rpmbuild-openshift-ansibleCB743M/BUILDROOT/openshift-ansible-3.9.1-1.git.0.a37a9af.el7.x86_64/usr/share/ansible/openshift-ansible/
cp: cannot stat 'files': No such file or directory
error: Bad exit status from /var/tmp/rpm-tmp.dyIJW8 (%install)
```

**Comment by sdodson** [comment]
Yeah, revert that commit and this should be fixed.

**Comment by mtnbikenc** [comment]
@yocum137 Let me know if you need assistance reverting 8337b187f43aa0df233b51740f6566e19f5c45d7
/lgtm cancel

**Pull request merged**